pkgs/development/tools/ocaml/cppo/default.nix

```diff
@@ -7,11 +7,15 @@ assert stdenv.lib.versionAtLeast ocaml.version "3.12";
 
 let param =
   if stdenv.lib.versionAtLeast ocaml.version "4.02" then {
-    version = "1.6.5";
-    sha256 = "03c0amszy28shinvz61hm340jz446zz5763a1pdqlza36kwcj0p0";
+    version = "1.6.6";
+    sha256 = "1smcc0l6fh2n0y6bp96c69j5nw755jja99w0b206wx3yb2m4w2hs";
     buildInputs = [ dune ];
     extra = {
-      inherit (dune) installPhase;
+      installPhase = ''
+        runHook preInstall
+        ${dune}/bin/dune install --prefix=$out --libdir=$OCAMLFIND_DESTDIR
+        runHook postInstall
+      '';
```

Comment:
I don't understand these changes. What are they meant to fix? If running `dune install` is generally better than running `opaline`, why not change this in the Dune derivation itself?

Reply:
I simply prefer to use dune install over opaline. I recall trying to set it to into `buildDunePackage` but there were some failing packages, but I haven't investigated much. 
I'll try to set into `buildDunePackage` it in another PR